src/user/relationship-model.md

```diff
@@ -1,21 +1,25 @@
-# Relationship model { #relationship_model } 
+# Relationship model { #relationship_model }
```

Comment:
I think we should add a diagram to the model. You adding a diagram to the PR made it easier to understand than all of the text/explanations you gave before. I might not be the only person which needs some visual 😅 

Something like https://graphviz.org/ or https://d2lang.com/ is useful so anyone can adapt and regenerate the image.

Reply:
I agree, I will have a look at this tomorrow but I will probably ask for some help as I am not sure how this work in the markdown docs.
In the meantime, if the rest is ok I would already merge it and move the ticket to test.